README.md

```diff
@@ -1,28 +1,9 @@
-<a id="markdown-transportd---http-middleware-as-a-service" name="transportd---http-middleware-as-a-service"></a>
 # transportd - HTTP Middleware As A Service
 
 *Status: Incubation*
 
-<!-- TOC -->
-
-- [transportd - HTTP Middleware As A Service](#transportd---http-middleware-as-a-service)
-    - [Overview](#overview)
-    - [Configuration](#configuration)
-        - [Runtime Settings](#runtime-settings)
-        - [Backend Settings](#backend-settings)
-        - [Route Settings](#route-settings)
-        - [Environment Variables](#environment-variables)
-    - [Custom Plugins And Builds](#custom-plugins-and-builds)
-        - [Writing A Component](#writing-a-component)
-        - [Generating A Build](#generating-a-build)
-    - [Using As A Library](#using-as-a-library)
-    - [Contributing](#contributing)
-        - [License](#license)
-        - [Contributing Agreement](#contributing-agreement)
-
-<!-- /TOC -->
-
-<a id="markdown-overview" name="overview"></a>
+<!-- TOC -->autoauto- [transportd - HTTP Middleware As A Service](#transportd---http-middleware-as-a-service)auto    - [Overview](#overview)auto    - [Configuration](#configuration)auto        - [Runtime Settings](#runtime-settings)auto        - [Backend Settings](#backend-settings)auto        - [Route Settings](#route-settings)auto        - [Environment Variables](#environment-variables)auto    - [Custom Plugins And Builds](#custom-plugins-and-builds)auto        - [Writing A Component](#writing-a-component)auto        - [Generating A Build](#generating-a-build)auto    - [Using As A Library](#using-as-a-library)auto    - [Contributing](#contributing)auto        - [License](#license)auto        - [Contributing Agreement](#contributing-agreement)autoauto<!-- /TOC -->
```

Comment:
Task: this appears to have been changed to a single line, which is much less readable.

Reply:
I have no idea how that happened... `dep` I guess?  Anyway, I'll revert it.